src/i16/i16vec2.rs

```diff
@@ -14,8 +14,15 @@ pub const fn i16vec2(x: i16, y: i16) -> I16Vec2 {
 }
 
 /// A 2-dimensional vector.
-#[derive(Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(Clone, Copy)]
+#[cfg_attr(
+    feature = "zerocopy",
+    derive(zerocopy::FromBytes, zerocopy::Immutable, zerocopy::KnownLayout)
+)]
+#[cfg_attr(not(feature = "zerocopy"), derive(PartialEq, Eq, Hash))]
```

Comment:
Why are these excluded if using zerocopy? Do the zerocopy traits inherit from these?